tests/test_version.py

```diff
@@ -0,0 +1,18 @@
+from unittest import TestCase
+
+import flask_audit_log
+
+
+class TestVersion(TestCase):
+
+    def test_version_exists(self):
+        self.assertTrue(hasattr(flask_audit_log, '__version__'))
+
+    def test_version_type(self):
+        self.assertTrue(isinstance(flask_audit_log.__version__, str))
+
+    def test_semver(self):
+        major, minor, patch = flask_audit_log.__version__.split(".")
+        self.assertTrue(major.isnumeric())
+        self.assertTrue(minor.isnumeric())
+        self.assertTrue(patch.isnumeric())
```

Comment:
Deze test klopt niet helemaal. Pre-releases kunnen een postfix hebben zoals rc (release candidate), alpha, beta, etc. Zie https://semver.org/. 

Zoals je de version nu split zouden die waardes in de `patch` terechtkomen, welke zou kunnen falen voor geldige semvers.

Reply:
Test was copied from the django library ;) But changed to use a regex to check the version on being a valid semver